clang/runtime/dpct-rt/include/dpct/blas_gemm_utils.hpp

```diff
@@ -100,6 +102,8 @@ class matrix_layout_t {
   std::uint64_t _rows;
   std::uint64_t _cols;
   std::int64_t _ld;
+  std::uint64_t batch_count;
```

Comment:
Since the value is not set in the ctor, I think giving it an init value is better.
```suggestion
  std::uint64_t batch_count = 1;
```

Reply:
OK